bootstrap-infra.py

```diff
@@ -1,13 +1,72 @@
 #!/usr/bin/env python3
+import os
+import random
+import string
 
 import boto3
+from botocore.exceptions import ClientError
+from dotenv import load_dotenv
 
 
 def main():
 
-    client = boto3.client('ec2')
-    output = client.describe_instances()
-    print(output)
+    def random_string(string_length):
+        """
+        Generating random string which is used as a group name, when creating a new security group.
+        :param string_length: (int) String length to be generated. For example 3 for 'abc'
+        :return: Random string
+        """
+        letters = string.ascii_letters
+        return ''.join(random.choice(letters) for i in range(string_length))
+
+    load_dotenv()
+    ec2 = boto3.client('ec2')
+    security_groups = os.getenv("SECURITY_GROUPS")
+    group_permissions = [
+        {'IpProtocol': 'tcp',
+         'FromPort': 5000,
+         'ToPort': 5000,
+         'IpRanges': [{'CidrIp': '0.0.0.0/0'}]},
+        {'IpProtocol': 'tcp',
+         'FromPort': 22,
+         'ToPort': 22,
+         'IpRanges': [{'CidrIp': '0.0.0.0/0'}]}
+    ]
```

Comment:
json object should be formatted like in the following example:
https://realpython.com/python-json/

Reply:
done